nengo_viz/server.py

```diff
@@ -43,8 +57,22 @@ def swi_favicon_ico(self):
         icon = pkgutil.get_data('nengo_viz', 'static/favicon.ico')
         return ('image/ico', icon)
 
+    def create_login_form(self):
+        if self.attempted_login:
+            message = 'Invalid password.  Try again.'
```

Comment:
Single space after period (maybe would get rendered well with HTML, but looks weird in fixed-width font)